ui/src/app/views/workspace-edit/component.ts

```diff
@@ -330,4 +330,11 @@ export class WorkspaceEditComponent implements OnInit {
   set workspacePopulationDetails(val) {
     this.workspace.researchPurpose.underservedPopulationDetails = val;
   }
+
+  get notValid() {
+    if (this.savingWorkspace) {
```

Comment:
Is this to prevent the user from clicking twice?

nit: May want to rename `notValid()` -> `allowSave()` or something. Technically an ongoing save shouldn't change the validity of a workspace. Also avoiding the negative in this method seems more natural to me (even if you negate everywhere it's used).

Reply:
Yes, it is to prevent clicking twice. I'll change to allowSave